src/core/packages/saved-objects/server-internal/src/object_types/index.ts

```diff
@@ -8,3 +8,7 @@
  */
 
 export { registerCoreObjectTypes } from './registration';
+
+// set minimum number of registered saved objects to ensure no object types are removed after 8.8
+// declared in internal implementation exclicilty to prevent unintended changes.
+export const MIN_SAVED_OBJECT_TYPES_COUNT = 127 as const;
```

Comment:
Exporting the constant from an internal package on purpose to avoid messing with the public types.

Reply:
done in https://github.com/elastic/kibana/pull/207142/commits/5dc727a05cc408145efab606ffa7943285769468